pkg/security/probe/probe_linux.go

```diff
@@ -321,16 +321,17 @@ func (p *Probe) handleAnomalyDetection(event *model.Event) {
 		return
 	}
 
-	if !profile.IsAnomalyDetectionEvent(event.GetEventType()) {
+	if profile.IsAnomalyDetectionEvent(event.GetEventType()) {
```

Comment:
why this `if` and the `else` ? why not using as previously the `IsAnomalyDetectionEvent` event ?

Reply:
The goal was to better differentiate the dedicated anomaly detection event types generated in kernel space (this is a hardcoded list of events) from the user configurable list of events that can generate anomalies. The first one is also used to implement some fast paths such as in `LookupEventInProfiles`.

Last but not least, I think there is value in making this list dynamic and per profile (from the configurable default) in case we want to control the "load" of anomalies by selectively preventing a workload from generating anomalies for specific event types; or simply because the content of a profile do not allow for generating anomalies on some event types (we're missing this parameter in profiles right now and we should add it otherwise the transition between profiles with `chmod` support and without `chmod` support will be messy for example).

Does it make sense ?